### PR TITLE
[ENH] Added support for logits or probabilities to the `WKLoss`

### PR DESCRIPTION
**Pull request by victormvy**
Previously, `WKLoss` required a probabilistic input to work properly. Given that, in PyTorch, models usually output logits, I have included the `use_logits` parameter to support logit inputs. When this parameter is set to `True`, the softmax of the input is computed before computing the loss. If `False`, the raw inputs are used.

### CODE REVIEW

**Comment by victormvy** [comment]
I tried to optimise the softmax computation integrating it in the WKLoss computation. Thus, instead of computing the softmax of `y_pred`, only the `max(y_pred)` and the `exp(y_pred)` had to be calculated. However, given that the softmax implementation of pytorch is highly optimised, that optimisation is not worth.

**Review by franberchez — APPROVED** [review]
All is great! Now is possible to use the loss function with probits or logits.

Thanks Victor!
